release/av/av.eo.esperanto/av.eo.esperanto.kpj

```diff
@@ -0,0 +1,99 @@
+      <Filepath>source\av.eo.esperanto.model.ts</Filepath>
+      <FileVersion>1.0</FileVersion>
+      <FileType>.ts</FileType>
+    </File>
+    <File>
+      <ID>id_be25819628403fa4f0a2f652eba09d93</ID>
+      <Filename>av.eo.esperanto.model.kps</Filename>
+      <Filepath>source\av.eo.esperanto.model.kps</Filepath>
+      <FileVersion>1.0</FileVersion>
+      <FileType>.kps</FileType>
+      <Details>
+        <Name>Esperanto</Name>
+        <Copyright>© Andrea Vaccari</Copyright>
+        <Version>1.0</Version>
+      </Details>
+    </File>
+    <File>
+      <ID>id_ede98e4633e239f933cbfd1f4e1b766c</ID>
+      <Filename>HISTORY.md</Filename>
+      <Filepath>HISTORY.md</Filepath>
+      <FileVersion></FileVersion>
+      <FileType>.md</FileType>
+    </File>
+    <File>
+      <ID>id_53e892b8b41cc4caece1cfd5ef21d6e7</ID>
+      <Filename>LICENSE.md</Filename>
+      <Filepath>LICENSE.md</Filepath>
+      <FileVersion></FileVersion>
+      <FileType>.md</FileType>
+    </File>
+    <File>
+      <ID>id_0730bb7c2e8f9ea2438b52e419dd86c9</ID>
+      <Filename>README.md</Filename>
+      <Filepath>README.md</Filepath>
+      <FileVersion></FileVersion>
+      <FileType>.md</FileType>
+    </File>
+    <File>
+      <ID>id_72bea735027a6e7807041c6432bbd10a</ID>
+      <Filename>av.eo.esperanto.model_info</Filename>
+      <Filepath>av.eo.esperanto.model_info</Filepath>
+      <FileVersion></FileVersion>
+      <FileType>.model_info</FileType>
+    </File>
+    <File>
+      <ID>id_35bd2a2e159f29af3798f1d4f05cd195</ID>
+      <Filename>av.eo.esperanto.model.js</Filename>
+      <Filepath>source\..\build\av.eo.esperanto.model.js</Filepath>
+      <FileVersion></FileVersion>
+      <FileType>.js</FileType>
+      <ParentFileID>id_be25819628403fa4f0a2f652eba09d93</ParentFileID>
+    </File>
+    <File>
+      <ID>id_356e5d149c1e539356d72698c1e401a6</ID>
+      <Filename>welcome.htm</Filename>
+      <Filepath>source\welcome.htm</Filepath>
+      <FileVersion></FileVersion>
+      <FileType>.htm</FileType>
+      <ParentFileID>id_be25819628403fa4f0a2f652eba09d93</ParentFileID>
+    </File>
+    <File>
+      <ID>id_8da344c4cea6f467013357fe099006f5</ID>
+      <Filename>readme.htm</Filename>
+      <Filepath>source\readme.htm</Filepath>
+      <FileVersion></FileVersion>
+      <FileType>.htm</FileType>
+      <ParentFileID>id_be25819628403fa4f0a2f652eba09d93</ParentFileID>
+    </File>
+    <File>
+      <ID>id_021fa3f70f65ac6dc532fe359f504b0c</ID>
+      <Filename>flag.png</Filename>
+      <Filepath>source\..\..\..\..\..\keyboards\release\e\eo_plus\source\flag.png</Filepath>
+      <FileVersion></FileVersion>
+      <FileType>.png</FileType>
+      <ParentFileID>id_be25819628403fa4f0a2f652eba09d93</ParentFileID>
+    </File>
+    <File>
+      <ID>id_0bf1020128a0de92f883ca3c3e61344b</ID>
+      <Filename>splash.png</Filename>
+      <Filepath>source\..\..\..\..\..\keyboards\release\e\eo_plus\source\splash.png</Filepath>
```

Comment:
I think the paths to flag.png and splash.png can point to files you've attached on this PR.
e.g. source\flag.png and source\splash.png.

Our CI build doesn't have the keyboards repo available to reference files there.

Reply:
These also aren't really needed in the .kpj file. But the paths in av.eo.esperanto.model.kps should be fixed.